include/libopencm3/dispatch/nvic.h

```diff
@@ -76,6 +76,12 @@
 #elif defined(MSP432E4)
 #   include <libopencm3/msp432/e4/nvic.h>
 
+#elif defined(NRF51)
+#	include <libopencm3/nrf/51/nvic.h>
+
+#elif defined(NRF52) || defined(NRF52FP)
```

Comment:
we don't need this || FP bit anymore right? we've dropped all that?

Reply:
Yeah, nrf51 is Cortex-M0, so no FP, nRF52 is implicitly FP